config.py

```diff
@@ -14,6 +14,11 @@ class BaseConfig(object):
     JWT_ACCESS_TOKEN_EXPIRES = timedelta(weeks=1)
     JWT_REFRESH_TOKEN_EXPIRES = timedelta(weeks=4)
 
+    # Aws details
+    AWS_ACCESS_KEY_ID = os.getenv('AWS_ACCESS_KEY_ID')
```

Comment:
```suggestion
    AWS_ACCESS_KEY_ID = os.getenv('AWS_ACCESS_ID')
```

Can you please change the expected environment variable to `AWS_ACCESS_ID`. I'm saying this because during the Community Open Session with @MayBurgos and @m-murad that Travis CI has that variable like `AWS_ACCESS_ID` instead of `AWS_ACCESS_KEY_ID`.